File: CSharp 1/HW.OperatorsExpressionsStatements/08.TrapezoidArea/TrapezoidArea.cs
using System;
 

namespace TrapezoidArea
{
    class TrapezoidArea
    {
        static void Main( )
        {
            //Write an expression that calculates trapezoid's area by given sides a and b and height h.

            Console.WriteLine("Enter trapezoid's side A");
            double sideA = double.Parse(Console.ReadLine());
            Console.WriteLine("Enter trapezoid's side B");
            double sideB = double.Parse(Console.ReadLine());
            Console.WriteLine("Enter trapezoid's height");
            double h = double.Parse(Console.ReadLine());

            double area = ((sideA + sideB) / 2) * h; //formula for trapezoid's area
            Console.WriteLine("Trapezoid's area is: " + area);

        }
    }
}

File: JS Fundamentals/JS_Operators_and_Expressions_HW/JS_Operators_and_Expressions_HW/Task_08/Trapezoid_Area.js
/// <reference path="c:\Users\M.Tonkov\Documents\Visual Studio 2013\Projects\JS_Operators_and_Expressions_HW\JS_Operators_and_Expressions_HW\js-console.js" />

//Write an expression that calculates trapezoid's area by given sides a and b and height h

function calculcate() {
    var sideA = parseFloat(document.getElementById('user-input-sideA').value),
        sideB = parseFloat(document.getElementById('user-input-sideB').value),
        height = parseFloat(document.getElementById('user-input-sideH').value),
        area;

    area = (( sideA + sideB )/ 2) * height;

    jsConsole.writeLine("Trapezoid's area is: " + area);
}
